content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -449,6 +451,9 @@ export const getServerSideProps: GetServerSideProps<
     );
   }
 
+  const filterByConceptId = serverData.toggles.conceptsById.value;
+  const queryParams = filterByConceptId ? queryParamsById : queryParamsByLabel;
```

Comment:
Any value that isn't undefined or empty string will return `true` here, even if its value is `"false"`, so I'd do

```suggestion
  const filterByConceptId = serverData.toggles.conceptsById.value === "true";
  const queryParams = filterByConceptId ? queryParamsById : queryParamsByLabel;
```

Reply:
The type system seems to indicate that these values are  `boolean | undefined`. I think using the `=== "true"` comparison will result in this always being false.

Reply:
Think you're right https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/server-data/toggles.ts#L51 !

I don't know where I had to enter it manually, maybe it was a cookie consent work of some kind...